frappe/public/js/frappe/web_form/web_form_list.js

```diff
@@ -201,7 +201,7 @@ export default class WebFormList {
 			() => (window.location.href = window.location.pathname + "?new=1")
 		);
 
-		if (this.rows.length <= this.page_length) {
+		if (this.rows.length >= this.page_length) {
```

Comment:
```suggestion
		if (this.rows.length > this.page_length) {
```
@Mangesh-Khairnar this should be the check right?

Reply:
Default list length is 20, whenever it reaches capacity it should allow user to load more rows. If it is set to be visible for more than 20 rows, it will never be shown to the user.

Reply:
But suppose there are 20 rows and the page length is also 20 then the "More" button should not be visible.